test/docs/OpenJ9TestUserGuide.md

```diff
@@ -262,12 +262,18 @@ target
     ```
     element that you want to exclude.
 
-    For tests that are disabeld with playlist.xml, `disabled.mk` is generated when running `make -f run_configure.mk`. It contains a list of disabled test targets. These targets can be executed through specifying the test target or add `disabled` in front of regular target.
+    For tests that are disabeld with playlist.xml, disabled tests lists are generated when running `make -f run_configure.mk`. These targets can be executed through specifying the test target or add `disabled` in front of regular target.
     ```    
         make _testA
         make _disabled.sanity.functional
         make _disabled.extended
     ```
+    Disabled tests and reasons can also be printed through adding `echo.disabled` in front of regular target.
+    ```    
+        make _echo.disabled.testA
+        make _echo.disabled.sanity.functional
+        make _echo.disabled.extended
+    ```
```

Comment:
There are no disabled tests lists anymore. I think we can keep the user guide's description the same as the quick start.